runtime/vm/ContinuationHelpers.cpp

```diff
@@ -1225,8 +1225,16 @@ takeVirtualThreadListToUnblock(J9VMThread *currentThread)
 			bool hasPlatformThreadWaiting = false;
 			while (NULL != current) {
 				bool unblocked = false;
+				bool isUnownedFlatlock = false;
 				J9ObjectMonitor *syncObjectMonitor = current->objectWaitMonitor;
 				U_32 state = J9VMJAVALANGVIRTUALTHREAD_STATE(currentThread, current->vthread);
+				j9object_t continuationObject = J9VMJAVALANGVIRTUALTHREAD_CONT(currentThread, current->vthread);
+				j9object_t blockingObject = J9VMJDKINTERNALVMCONTINUATION_BLOCKER(currentThread, continuationObject);
+				if (LN_HAS_LOCKWORD(currentThread, blockingObject)
```

Comment:
Do we need to handle the case that `LN_HAS_LOCKWORD()` is false ?

https://github.com/eclipse-openj9/openj9/blob/7bae41731b0b1f9bf38069053261513b5de6d33b/runtime/vm/monhelpers.c#L45-L87

FLC is checked here on `count` which comes from either `alternateLockword` or `J9OBJECT_MONITOR_EA()`.

Reply:
let me check, I though for objects without lockword, we would have the used the inflated monitor by default for vthread yield